elasticsearch/config/load_settings.sh

```diff
@@ -27,22 +100,65 @@ done
 if [ $ENABLE_CONFIGURE_S3 ]; then
   #Wait for Elasticsearch to be ready to create the repository
   sleep 10
-  IP_PORT="${ELASTICSEARCH_IP}:${ELASTICSEARCH_PORT}"
 
-  if [ "x$S3_PATH" != "x" ]; then 
+  if [ "x$S3_PATH" != "x" ]; then
 
-    if [ "x$S3_ELASTIC_MAJOR" != "x" ]; then 
-      ./config/configure_s3.sh $IP_PORT $S3_BUCKET_NAME $S3_PATH $S3_REPOSITORY_NAME $S3_ELASTIC_MAJOR 
+    if [ "x$S3_ELASTIC_MAJOR" != "x" ]; then
+      bash /usr/share/elasticsearch/config/configure_s3.sh $el_url $S3_BUCKET_NAME $S3_PATH $S3_REPOSITORY_NAME $S3_ELASTIC_MAJOR
 
     else
-      ./config/configure_s3.sh $IP_PORT $S3_BUCKET_NAME $S3_PATH $S3_REPOSITORY_NAME 
+      bash /usr/share/elasticsearch/config/configure_s3.sh $el_url $S3_BUCKET_NAME $S3_PATH $S3_REPOSITORY_NAME
 
     fi
 
   fi
 
 fi
 
+##############################################################################
+# Setup passwords for Elastic Stack users
+##############################################################################
+
+if [[ $SECURITY_ENABLED == "yes" ]]; then
+  MY_HOSTNAME=`hostname`
+  echo "Hostname:"
+  echo $MY_HOSTNAME
+  if [[ $SECURITY_EXPECTED_HOSTNAME == $MY_HOSTNAME ]]; then
+    echo "Setting up passwords for all Elastic Stack users"
+
+    sleep 10
```

Comment:
I think it is not necessary since the code below is waiting for Elastic.

Reply:
True, I miss remove it.